### PR TITLE
fix #60952 news.noteru.com

### LINKED ISSUE #60952: news.noteru.com
### Issue URL (Annoyance)
 [https://news.noteru.com/dlinnye-volosy-kak-legko-otrastit-kosu-do-poyasa-1115/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnews.noteru.com%2Fdlinnye-volosy-kak-legko-otrastit-kosu-do-poyasa-1115%2F)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by stanislav-atr**
#60952

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -124,6 +124,7 @@ pcland.hu##.HMi-popup-overlay
 dictionary.com##.marketing-message
 thesaurus.com##div[class*="-RightMarketingSlot"]
 gastronom.ru##.js-bother
+news.noteru.com##.popup
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:127]: It is push notification, it's better to block script which is created it.
And also block gogool.ru, push management service

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt:127]: yep. Add to `/firebase-messaging.js` rule

**Comment by Sergey-Lyapin** [comment]
What about  these rules?
`||news.noteru.com/wp-content/themes/noteru/assets2/js/push.js`
`||gogool.ru^$third-party`

**Review by Alex-302 — APPROVED** [review]

**Comment by Sergey-Lyapin** [comment]
Please add domain to this rule `/push.js$domain=acunn.com|tsn.ua|sputnik.by|nation-news.ru` and this rule `||gogool.ru^$third-party` in new pull request

**Pull request merged**